js/modules/k6/http/request.go

```diff
@@ -73,6 +73,61 @@ func (c *Client) Request(method string, url goja.Value, args ...goja.Value) (*Re
 	return c.responseFromHTTPext(resp), nil
 }
 
+// Request makes an http request of the provided `method` and returns a corresponding response by
+// taking goja.Values as arguments
+func (c *Client) asyncRequest(method string, url goja.Value, args ...goja.Value) (*goja.Promise, error) {
```

Comment:
Let's update the doc comment.

Reply:
[0834272](https://github.com/grafana/k6/pull/2877/commits/0834272dec6963d5f4e6432d0250ccc75722936b)